ZenPacks/zenoss/LinuxMonitor/impact.py

```diff
@@ -19,6 +19,8 @@
 from Products.ZenModel.CPU import CPU
 from Products.ZenModel.FileSystem import FileSystem
 
+from ZenPacks.zenoss.LinuxMonitor.HardDisk import HardDisk
```

Comment:
Now that the impacts within zenpack.yaml are defined as DynamicView impacts/impacted_by relationships, these corresponding relationships should probably also be defined via DynamicView adapters instead of Impact adapters.